giantswarm/management-clusters-fleet
# Bumping Flux version

## Description

This PR introduces the `v0.27.3` version of Flux. 

## Testing Procedure

1. Switch some test installation to this branch (testing)
2. Switch our `gollum` and `gorilla` installations to this branch (dog fooding)
3. After testing and dog fooding, suspend `gollum`, `gorilla` and `gauss` clusters
4. Resume `main` on resources from this PR and make it ready to review
5. Notify customers about the update, backup and suspend their reources
6. Merge, but do not remove the branch to not break the above installations
7. Switch back the installations from steps 1 & 2 to the main branch
8. Remove the old branch
9. Resume customer resources and make sure they work
